libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -56,6 +56,10 @@
 
   private static final String GENERIC_CONTROL_ATTR = "*";
 
+  /** Default width and height for VP9. */
+  private static final int DEFAULT_VP9_WIDTH = 320;
```

Comment:
For VP8, we ended up with a javadoc like this, please do the same here.

```
  /**
   * Default width for VP8.
   *
   * <p>RFC7741 never uses codec specific data (like width and height) in the fmtp attribute. These
   * values are taken from <a
   * href=https://cs.android.com/android/platform/superproject/+/master:frameworks/av/media/codec2/components/vpx/C2SoftVpxDec.cpp;drc=749a74cc3e081c16ea0e8c530953d0a247177867;l=70>Android's
   * software VP8 decoder</a>.
   */
```

https://github.com/androidx/media/blob/f75710be933aca23f24c5b6d8f2f1925ba1da543/libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java#L93